mmpd-lib/src/midi.rs

```diff
@@ -119,8 +121,121 @@ fn parse_message(bytes: &[u8]) -> Option<MidiMessage> {
     }
 }
 
+/// Parses a string describing a note into MIDI note number(s)
+///
+/// key_str should be in the format "<note name>[accidental][octave]" where angle brackets denote
```

Comment:
Could also say "accidentals" here in plural form.
```suggestion
/// key_str should be in the format "<note name>[accidentals][octave]" where angle brackets denote
```